### DISCUSSION TITLE
4k HDR hdcp 2.2 grabber

### DISCUSSION THREAD

**Discussion by robons17** [awawa-dev/HyperHDR#481] [Q&A] [answered]
Hi, thanks for your job. I'm chosing materiale for mi first setup. 
I woukd ask which Is a good budget grabber today.
No preference for splitter or loop.
Thanks

**Comment by awawa-dev** [comment, marked as answer]
Hi
To my knowledge there is currently no hdcp 2.2 capable grabber with a loop. Without a loop there is probably e.g. Ezcap 331 https://www.hyperhdr.eu/2022/01/ezcap-331-short-review-some-info-about.html but it's not recommended (its EDID doesn't support HDR anyway). So if hdcp 2.2 is important, you should consider only external HDMI splitter solution and a grabber of your choice.

**Reply by robons17** [reply]
Thanks. I think i buy EZ-SP12H2 as you suggest. (Or something by feintech).
Can i ask you an opinioni on the grabber ? Condidering also the LUT calibration? There are so manu clone that Is very diffucult for a noob...

**Reply by awawa-dev** [reply]
Most users are happy with ms2109 (this is USB2.0 grabber....do not choose "USB3.0" fake version, the picture of my ms2109 grabber is on my blog, choose model that looks identical). If you want faster USB3.0 model then maybe Ezcap 321? It seems to me that I put all the useful information about specific grabbers on the blog.

**Reply by robons17** [reply]
Yes, i  have read all about grabbers. The only doubt i have Is that all msi2109 i found says: max input res 4k30hz. Thanks .

**Reply by awawa-dev** [reply]
Good HDMI splitter can scale down 4k to 1080p for ms2109.

**Comment by vkodsi** [comment]
For instance https://www.easycoolav.com/products/hdmi-splitter-4k-60hz-arc-earc-for-soundbar-hdmi-switch-1-in-2-out-2-input-to-2-output ;)
